docs/conditions.md

```diff
@@ -20,6 +22,8 @@ Allows a flag to be enabled for the username given as the condition's value.
 FLAGS = {'MY_FLAG': {'user': 'jane.doe'}}
 ```
 
+The value can be given as a Python `True` or `False` object or as the strings `"true"`, `"True"`, `"False"`, or `"false"`.
```

Comment:
Mistaken addition?

Reply:
Nope.

Reply:
I don't understand why you would add this under the `user` condition.